include/netinet/tcp.h

```diff
@@ -16,6 +16,8 @@
 #ifndef _LIBPHOENIX_NETINET_TCP_H_
 #define _LIBPHOENIX_NETINET_TCP_H_
 
-#define TCP_NODELAY 1
+#define TCP_NODELAY   1
+#define TCP_KEEPIDLE  4
+#define TCP_KEEPINTVL 5
```

Comment:
I am concerned about those values. lwip defines them differently here: 
https://github.com/phoenix-rtos/lwip/blob/383bf4f3334f88668a05120e7b503e29158eb38c/src/include/lwip/sockets.h#L283
So a call to `setsockopt()` with `TCP_KEEPIDLE` or `TCP_KEEPINTVL` would actually do a different thing. Please check it.
You should either define them to be compliant with lwip or "translate" them in our lwip sockets port.

Reply:
Yes, I realise that azure does not require some specific values, so I think they should be compatible with those defined in lwip. It's corrected now.